test/helpers/kubectl.go

```diff
@@ -140,6 +140,7 @@ var (
 		"global.nodePort.mode":        "snat",
 		"global.gke.enabled":          "true",
 		"global.nativeRoutingCIDR":    "10.0.0.0/8",
+		"config.ipam":                 "kubernetes",
```

Comment:
FWIW in the current configmap, the `.Values.global.gke.enabled` also configures this. It's defined earlier in the file as well based on this `config.ipam` setting but I'm not sure about the precedence between the multiple declarations.

Reply:
Good point, I'll remove this and re-test now that I know how to deal with the namespaces stuck at "Terminating" state!

Reply:
Tested, does not work without `config.ipam: "kubernetes"`. Here's the diff on the CI generated cilium yaml, from with to without:
```
$ diff cilium-1619663866a24408.yaml cilium-1619670fe7897668.yaml
237a238
>   k8s-require-ipv6-pod-cidr: "false"
256c257,259
<   ipam: "kubernetes"
---
>   ipam: "cluster-pool"
>   cluster-pool-ipv4-cidr: "10.0.0.0/8"
>   cluster-pool-ipv4-mask-size: "24"
```